demos/elevation.scss

```diff
@@ -18,3 +18,7 @@
 
 @import "../packages/mdc-elevation/mdc-elevation";
 @import "../packages/mdc-typography/mdc-typography";
+
+.demo-elevation--custom-color {
+  @include mdc-elevation(16, $color: secondary-dark, $opacity-boost: .1);
```

Comment:
Nit: why did you include the variable names here, when you're specifying all of them, in order, anyway?

Reply:
I could probably remove the `$color` name since it's pretty obvious what `secondary-dark` is. But for `$opacity-boost`, I think it's more readable to have the param name; without it, `.1` is just a magical number that means who-knows-what, and you have to go read the documentation or source code to figure out what it does.

Reply:
Removed `$color`!